lib/locales/pl.ts

```diff
@@ -230,7 +230,7 @@ export const locale: Locale = {
       'Św. Dionizego, biskupa i męczennika, patrona miasta i diecezji Saint-Denis',
     divine_mercy_sunday: '2 Niedziela Wielkanocna czyli Niedziela Miłosierdzia Bożego',
     dominic_de_guzman_priest: 'Św. Dominika, prezbitera',
-    easter_sunday: 'Niedziela Wielkanocna Zmartwychwstania Pańskiego',
+    easter_sunday: 'Niedziela Wielkanocy Zmartwychwstania Pańskiego',
```

Comment:
This sounds strange. I presume `Wielkanocy` should be omitted. Could you verify this please?

Reply:
The full, official name (as printed in the 2nd edition of the lectionary released in 2016, which includes linguistic changes that will be followed in the upcoming 3rd ed. of the Roman Missal) is `Niedziela Wielkanocy Zmartwychwstania Pańskiego`.

However, shorter version, used in the lectionary header (next to a page number) is `Niedziela Zmartwychwstania Pańskiego`.

The latter is commonly used, so I'm ok with using the non-official, but familiar-sounding name.

Reply:
Well, I prefer using official names to the common ones, therefore if `Niedziela Wielkanocy Zmartwychwstania Pańskiego` is the official name from the Lectionary based on the future MR (ed typ tertia), I am okay with that. I just wanted to be sure it is so, as to me (a Slovak), it sounds a bit strange; I would say either `Niedziela Wielkanocy` or `Niedziela Zmartwychwstania Pańskiego`, but I can’t speak Polish (just a bit; I mostly understand it).